src/main/java/uk/gov/hmcts/divorce/divorcecase/model/FinalOrder.java

```diff
@@ -168,6 +168,72 @@ public enum Granted implements HasLabel {
     )
     private YesOrNo finalOrderReminderSentApplicant2;
 
+
+    @CCD(
+        label = "Applicant 1 Can Intend To Switch To Sole at Final Order stage",
```

Comment:
not sure casing is correct should it be not Applicant can intend to switch to sole at Final Order stage